x/commitment/client/wasm/query_unstaked_positions.go

```diff
@@ -50,6 +50,12 @@ func (oq *Querier) BuildUnStakedPositionResponseCW(ctx sdk.Context, unbondingDel
 			// Get validator
 			val := oq.stakingKeeper.Validator(ctx, valAddress).(stakingtypes.Validator)
 			votingPower := sdk.NewDecFromInt(val.GetBondedTokens()).QuoInt(totalBonded).MulInt(sdk.NewInt(100))
+
+			website := val.Description.Website
+			if len(website) < 1 {
+				website = " "
```

Comment:
Can we probably remove website empty check and let it be fine with empty website rather than putting space? I think it's not a good approach to bypass the check with `" "`

Reply:
@jelysn same above.